src/tests/test_sqlthread.py

```diff
@@ -5,40 +5,237 @@
 import threading
 import unittest
 
-from .common import skip_python3
-
-skip_python3()
 
 os.environ['BITMESSAGE_HOME'] = tempfile.gettempdir()
 
-from pybitmessage.helper_sql import (
-    sqlQuery, sql_ready, sqlStoredProcedure)  # noqa:E402
-from pybitmessage.class_sqlThread import sqlThread  # noqa:E402
+from pybitmessage.class_sqlThread import TestDB  # noqa:E402
 from pybitmessage.addresses import encodeAddress  # noqa:E402
 
 
-class TestSqlThread(unittest.TestCase):
+def filter_table_column(schema, column):
+    """
+        Filter column from schema
+    """
+    for x in schema:
+        for y in x:
+            if y == column:
+                yield y
+
+
+class TestBitmessageDB(unittest.TestCase):  # pylint: disable=protected-access
     """Test case for SQL thread"""
 
+    # query file path
+    __name__ = None
+    root_path = os.path.dirname(os.path.dirname(__file__))
+    test_db = None
+    conn = None
+    cur = None
+
     @classmethod
-    def setUpClass(cls):
-        # Start SQL thread
-        sqlLookup = sqlThread()
-        sqlLookup.daemon = True
-        sqlLookup.start()
-        sql_ready.wait()
+    def setUp(cls):
+        """
+            Drop all tables before each test case start
+        """
+        cls.test_db = TestDB()
+        cls.conn, cls.cur = cls.test_db._connection_build()  # pylint: disable=protected-access
```

Comment:
why do you need this? how about just using `cls.test_db.conn` and `cls.test_db.cur`?